### ISSUE TITLE
In projects with many branches, the branch tree display is not compact enough

### ISSUE THREAD

**Issue by MaiLunJiye** [mlange-42/git-graph#69]
```
git-graph -m simple  -s r
```

![image](https://user-images.githubusercontent.com/20548127/230898824-992970a0-7167-4776-8d5b-b34bc6f04860.png)

```
git log --all --decorate --oneline --graph
```
![image](https://user-images.githubusercontent.com/20548127/230899047-4dfe1b9b-f8ea-4c8c-9cea-a67937c5ac10.png)

**Comment by mlange-42** [comment]
@peso Exactly, it is not a bug. It might still be possible to improve the rendering. However, this would result in branches moving to other columns when scrolling futher down. So not sure...
